src/pages-and-resources/discussions/data/slice.js

```diff
@@ -42,6 +44,10 @@ const slice = createSlice({
       const { status } = payload;
       state.saveStatus = status;
     },
+    updateValidationStatus: (state, { payload }) => {
```

Comment:
Can we add a test for `updateValidationStatus`?

Reply:
yes. we can .